crates/biome_html_analyze/Cargo.toml

```diff
@@ -19,6 +19,7 @@ biome_diagnostics        = { workspace = true }
 biome_html_factory       = { workspace = true }
 biome_html_syntax        = { workspace = true }
 biome_rowan              = { workspace = true }
+biome_rule_options       = { workspace = true }
```

Comment:
This is added to fix this error

![Image](https://github.com/user-attachments/assets/e57b3b9d-558c-40f0-8a46-97c890eee2b8)